src/main/scala/io/gatling/sbt/settings/gatling/RecoverEnterprisePluginException.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2011-2022 GatlingCorp (https://gatling.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.gatling.sbt.settings.gatling
+
+import java.util.UUID
+
+import scala.jdk.CollectionConverters._
+import scala.util.{ Failure, Try }
+
+import io.gatling.plugin.exceptions._
+import io.gatling.plugin.model.Simulation
+
+import sbt.Configuration
+import sbt.internal.util.ManagedLogger
+
+class RecoverEnterprisePluginException(config: Configuration) {
+
+  protected def recoverEnterprisePluginException[U](logger: ManagedLogger): PartialFunction[Throwable, Try[U]] = {
+    case e: UnsupportedJavaVersionException =>
+      logger.error(s"""${e.getMessage}
+                      |In order to target the supported Java bytecode version, please use the following sbt settings:
+                      |scalacOptions += "-target:${e.supportedVersion}"
+                      |javaOptions ++= Seq("--release", "${e.supportedVersion}")
```

Comment:
Not sure about this one
```suggestion
                      |javacOptions ++= Seq("--release", "${e.supportedVersion}")
```